client/notices_test.go

```diff
@@ -42,6 +42,8 @@ func (cs *clientSuite) TestNotice(c *C) {
 	}}`
 	notice, err := cs.cli.Notice("123")
 	c.Assert(err, IsNil)
+	c.Assert(cs.req.Method, Equals, "GET")
```

Comment:
Is this just adding consistency to our tests?

Reply:
Yep, that's right -- and specifically that the client is sending the right HTTP method.